syntax = "proto3";

package iris_classifier;

service Predictor {
    rpc Predict (Sample) returns (Response);
}

message Sample {
    float sepal_length = 1;
    float sepal_width = 2;
    float petal_length = 3;
    float petal_width = 4;
}

message Response {
    string classification = 1;
}
